Doc/whatsnew/3.14.rst

```diff
@@ -371,6 +371,12 @@ Other language changes
   The testbed can also be used to run the test suite of projects other than
   CPython itself. (Contributed by Russell Keith-Magee in :gh:`127592`.)
 
+* :option:`-X importtime <-X>` now accepts value ``2``, which indicates that
```

Comment:
`2` seems fairly arbitrary, would you consider e.g. `-X importtime=include-cached` (or a variant)?

Reply:
this was already discussed in the accompanying thread and email chain--I feel the complexity of supporting an arbitrary string value doesn't really outweigh the marginal benefit of the verbosity, as this is a fairly niche tool that already relies on documentation to explain what it does